### DISCUSSION TITLE
[11 Settembre 2023] Pipeline

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/architettura-degli-elaboratori#86] [Pipeline (M-Z)]
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/10efbf96-732d-4490-86e8-e335adc439e3)
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/f2e96950-20cd-49f4-9ce5-fc85a886a68d)
![image](https://github.com/sapienzastudentsnetwork/archelab2223/assets/66442720/bb9ae295-bb0b-4881-b607-42db198bc288)

**Comment by rimaout** [comment]
<img width="696" alt="image" src="https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/assets/116072651/bc0cdfb8-4b09-4337-9dc0-2b99e2df101f">
